.travis.yml

```diff
@@ -2,11 +2,8 @@ sudo: false
 language: python
 cache: pip
 python:
-  - 2.7
-  - 3.4
   - 3.5
   - 3.6
-  - pypy
```

Comment:
Add 3.7 so you get all the deprecation warnings for 3.8

Reply:
done: 28e1ee75d3c03024964c45347cc3bf04a8db5c9a